python/hail/dataset.py

```diff
@@ -2946,7 +2946,57 @@ def same(self, other):
+        | ``nCalled``               | Int    | Sum of ``nHomRef`` + ``nHet`` + ``nHomVar``        |
+        +---------------------------+--------+----------------------------------------------------+
+        | ``nNotCalled``            | Int    | Number of uncalled variants                        |
+        +---------------------------+--------+----------------------------------------------------+
+        | ``nSNP``                  | Int    | Number of SNP variants                             |
+        +---------------------------+--------+----------------------------------------------------+
+        | ``nInsertion``            | Int    | Number of insertion variants                       |
+        +---------------------------+--------+----------------------------------------------------+
+        | ``nDeletion``             | Int    | Number of deletion variants                        |
+        +---------------------------+--------+----------------------------------------------------+
+        | ``nSingleton``            | Int    | Number of private variants                         |
+        +---------------------------+--------+----------------------------------------------------+
+        | ``nTransition``           | Int    | Number of transition (A-G, C-T) variants           |
+        +---------------------------+--------+----------------------------------------------------+
+        | ``nTransversion``         | Int    | Number of transversion variants                    |
+        +---------------------------+--------+----------------------------------------------------+
+        | ``nNonRef``               | Int    | Sum of ``nHet`` and ``nHomVar``                    |
+        +---------------------------+--------+----------------------------------------------------+
+        | ``rTiTv``                 | Double | Transition/Transversion ratio                      |
+        +---------------------------+--------+----------------------------------------------------+
+        | ``rHetHomVar``            | Double | Het/HomVar ratio across all variants               |
+        +---------------------------+--------+----------------------------------------------------+
+        | ``rInsertionDeletion``    | Double | Insertion/Deletion ratio across all variants       |
+        +---------------------------+--------+----------------------------------------------------+
+        | ``dpMean``                | Double | Depth mean across all variants                     |
+        +---------------------------+--------+----------------------------------------------------+
+        | ``dpStDev``               | Double | Depth standard deviation across all variants       |
+        +---------------------------+--------+----------------------------------------------------+
+        | ``gqMean``                | Double | The average genotype quality across all variants   |
```

Comment:
"Genotype quality mean across all variants" would be more parallel to the standard deviation one